@stellar/anchor-tests/src/tests/sep38/postQuote.ts

```diff
@@ -227,6 +247,24 @@ export const canCreateQuote: Test = {
     this.context.provides.sep38QuoteResponseObj = quoteResponse;
     return result;
   },
+});
+export const canCreateQuote: Test = {
+  sep: 38,
+  assertion: "returns a valid response (make sure all contexts pass)",
+  group: "POST /quote",
+  dependencies: (config: Config) => {
+    const result: Test[] = [returnsValidJwt, hasQuoteServer];
+    for (const sep38Context of config.sepConfig?.[38]?.contexts ?? []) {
+      result.push(canCreateQuoteWithContext(sep38Context));
+    }
+    return result;
+  },
+  context: { expects: {}, provides: {} },
+  failureModes: genericFailures,
+  async run(_config: Config): Promise<Result> {
+    const result: Result = { networkCalls: [] };
+    return result;
+  },
```

Comment:
What if the `canCreateQuoteWithContext` test made `POST /quote` calls for every context value specified in the config file, instead of adding a meta-style test like this?

Reply:
This applies to all the tests that use this pattern.

Reply:
Yeah, that's probably easier to maintain... updated that in https://github.com/stellar/stellar-anchor-tests/pull/87/commits/567b2f1d2b801cdc01499421661b7aa320d4a1ae.